src/Handler/ExceptionHandler.php

```diff
@@ -34,18 +36,26 @@ class ExceptionHandler
     private $whoops;
 
     /**
+     * @var LoggerInterface|null
+     */
+    private $logger;
+
+    /**
      * @param ExceptionHandlerPreferences $preferences
      * @param Negotiator $negotiator
      * @param ResolverInterface $resolver
      * @param Whoops $whoops
+     * @param LoggerInterface|null $logger
      */
     public function __construct(
         ExceptionHandlerPreferences $preferences,
         Negotiator $negotiator,
         ResolverInterface $resolver,
-        Whoops $whoops
+        Whoops $whoops,
+        LoggerInterface $logger = null
```

Comment:
This is good because it works with existing configurations, so we can include it in a minor release.